### ISSUE TITLE
Potion/Item quickbar?

### ISSUE THREAD

**Issue by ghost** [Svenzor84/Ancestors-Quest-2-Unity-Scripts#1]
@Svenzor84 At the top of the screen it might be helpful to have a quickbar with potions/usable items, the amount that you have, and the hotkey for using them. 
I know this info is available in the inventory, but would be nice info to have on-screen constantly, and it looks liek you've got the space for it.

**Comment by Svenzor84** [comment]
I was having some trouble finding the open screen-space to add an entire quickbar containing all usable items, as I don't want to block any part of the play area (rooms grow to take up the basically the entire screen late game).  But I really like the idea, so in compromise I added a non-clickable health potion counter in the top left hand corner of the screen (also depicts the hotkey tied to health potions, 1).  I would like to incorporate a more robust quickbar, like you mention, but in the meantime this will at least give the player a visualization of remaining health potions (the most used item) as well as a reminder of the hotkey.

**Comment by Svenzor84** [comment]
After delving into the Unity API a bit more, specifically the use of OnGUI(), I was able to come up with a solution that retains a majority of screen space and should be responsive. I have implemented a short 3 button "hotbar" with click/tap-able buttons tied to potion, strength potion, and experience potion that displays on the left hand side of the screen. The buttons also display the associated hotkey and inventory count for each item. These are the only hotkeys for now; higher level health potions are accessible only through the inventory panel. Leaving this issue open for testing after v1.3 is available.
